hopp/simulation/technologies/sites/site_info.py

```diff
@@ -203,71 +208,9 @@ def plot(self,
-        elif isinstance(self.polygon, MultiPolygon):
-            shape = self.polygon.geoms
-        for geom in shape:    
-            xs, ys = geom.exterior.xy    
-            plt.fill(xs, ys, alpha=0.3, fc='g', ec='none')
 
         plt.tick_params(which='both', labelsize=15)
         plt.xlabel('x (m)', fontsize=15)
         plt.ylabel('y (m)', fontsize=15)
 
         return figure, axes
-
-    def kml_write(self, filepath, turb_coords=None, solar_region=None, wind_radius=200):
-        if turb_coords is not None:
-            turb_coords = np.atleast_2d(turb_coords)
-            for n, (x, y) in enumerate(turb_coords):
-                self.append_kml_data(self.kml_data, Point(x, y).buffer(wind_radius), f"Wind Turbine {n + 1}")
-        if solar_region is not None:
-            if isinstance(solar_region, Polygon):
-                solar_region = [solar_region]
-            elif isinstance(solar_region, MultiPolygon):
-                solar_region = solar_region.geoms
-            for n, poly in enumerate(solar_region):
-                self.append_kml_data(self.kml_data, poly, f"Solar Region {n + 1}")
-        with open(filepath, 'w') as kml_file:
-            kml_str = self.kml_data.to_string(prettyprint=True)
-            kml_file.write(kml_str)
-
-    @staticmethod
-    def kml_read(filepath):
-        k = kml.KML()
-        with open(filepath) as kml_file:
-            k.from_string(kml_file.read().encode("utf-8"))
-        features = list(k.features())[0]
-        placemarks = list(list(features.features())[0].features())
-        
-        gmaps_epsg = pyproj.CRS("EPSG:4326")
-        project = None
-
-        valid_region = None
-        for pm in placemarks:
-            if "boundary" in pm.name.lower():
-                valid_region = make_valid(pm.geometry)
-                lon, lat = valid_region.centroid.x, valid_region.centroid.y
-                if project is None:
-                    zone_num = utm.from_latlon(lat, lon)[2]
-                    utm_proj= pyproj.CRS(f'EPSG:326{zone_num}')
-                    project = pyproj.Transformer.from_crs(gmaps_epsg, utm_proj, always_xy=True).transform
-                valid_region = transform(project, valid_region)
-                break
-        if valid_region is None:
-            raise ValueError("KML file needs to have a placemark with a name containing 'Boundary'")
-        for pm in placemarks:
-            if 'exclusion' in pm.name.lower():
-                try:
-                    valid_region = valid_region.difference(transform(project, pm.geometry.buffer(0)))
-                except:
-                    valid_region = valid_region.difference(transform(project, make_valid(pm.geometry)))
-        return k, valid_region, lat, lon
-
-    @staticmethod
-    def append_kml_data(kml_data, polygon, name):
-        folder = kml_data._features[0]._features[0]
-        new_pm = kml.Placemark(name=name)
-        new_pm.geometry = polygon
-        folder.append(new_pm)
```

Comment:
why was this removed? this is needed for reading the site boundary from a KML file